leveldbjni/src/main/java/org/fusesource/leveldbjni/internal/JniDB.java

```diff
@@ -219,59 +219,15 @@ public void suspendCompactions() throws InterruptedException {
         if( db==null ) {
             throw new DBException("Closed");
         }
-        db.suspendCompactions();
+	// Not implemented
+        // db.suspendCompactions();
     }
+
     public void resumeCompactions() {
         if( db==null ) {
             throw new DBException("Closed");
         }
-        db.resumeCompactions();
+	// Not implemented
+        // db.resumeCompactions();
     }
-
```

Comment:
For the code change here, if we don't invoke the functions below, do we still need to comment them out?

Reply:
Those are already commented out code in the original codebase, I deleted them since we do not have support for resume/suspend compactions. Should the comments be retained?